spec/requests/groups_controller_spec.rb

```diff
@@ -451,6 +451,11 @@ def expect_type_to_return_right_groups(type, expected_group_ids)
     it "should return the right response" do
       sign_in(user)
 
+      group.update!(
```

Comment:
For this feature, I was expecting more like an integration test that would 

1. create a public group that can't be mentioned (with at least one member)
2. create a post with an @mention to that group
3. ensure no mention notifications were sent to members of the group
4. ensure the mention gets wrapped in the cooked version of the post

Reply:
Each individual step is tested somewhere, cooked_post_processor_spec, post_alerter_spec or pretty_text_spec, but I agree that an integration test would be ideal, so I wrote one. :)